pkg/core/resources/model/rest/list.go

```diff
@@ -34,8 +34,8 @@ func (rec *ResourceListReceiver) UnmarshalJSON(data []byte) error {
 	if err := json.Unmarshal(data, &list); err != nil {
 		return err
 	}
-	rec.ResourceList.Total = list.Total
-	rec.ResourceList.Items = make([]Resource, len(list.Items))
+	rec.Total = list.Total
```

Comment:
is this equivalent to the previous one? 🤔

Reply:
yes